resources/compass/charts/gateway/templates/gateway.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Please see my comment in file `resources/compass/charts/gateway/templates/virtualservice.yaml` line 15.